string essid
uint8 link_quality
uint8 link_quality_max
int32 bitrate
